src/modules/mc_rate_control/mc_rate_control_params.c

```diff
@@ -241,8 +241,8 @@ PARAM_DEFINE_FLOAT(MC_YR_INT_LIM, 0.30f);
  * Yaw rate differential gain. Small values help reduce fast oscillations. If value is too big oscillations will appear again.
  *
  * @min 0.0
- * @decimal 2
- * @increment 0.01
+ * @decimal 4
```

Comment:
PR description says 3 decimals :open_mouth:

Reply:
Uh, copy paste typo in the PR description, sorry!